README.md

```diff
@@ -1,5 +1,19 @@
 ## Canvas Course Manager
```

Comment:
The readme should have documentation of LTI configuration in canvas. That will help with new people join the team

Reply:
Okay, I will add some info. However, I want to avoid parroting similar docs we've already written for other projects. I will add a paragraph describing the Canvas configuration and provide a sample LTI config in JSON.